codecov.yml

```diff
@@ -4,4 +4,4 @@ fixes:
 comment:
     # The code coverage is made up of 5 test runs so only after all coverage
```

Comment:
does this relate to removing job queue? also the comment above says 5 but should be 4?

Reply:
Fixed